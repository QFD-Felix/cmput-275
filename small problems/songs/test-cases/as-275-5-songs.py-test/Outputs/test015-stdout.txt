14 8 3 4 6 16 5 17 0 10 5 11 20 4 6 1 0 9 0 6 10 0 1 4 4 0 3 4 0 0 0 0 0 0
