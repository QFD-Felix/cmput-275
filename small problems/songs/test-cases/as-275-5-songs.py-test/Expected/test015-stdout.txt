14 3 3 4 6 4 5 17 0 10 5 11 18 4 6 1 5 9 0 6 12 0 1 4 4 0 5 6 0 0 0 3 0 0 0 0
